client/src/App.js

```diff
@@ -1,61 +1,60 @@
 import { Route, Switch } from 'react-router-dom';
-
-import styled from 'styled-components/macro';
 import { useEffect, useState } from 'react';
+
+import Home from './pages/Home';
+import Watchlist from './pages/Watchlist';
+import Library from './pages/Library';
+import Friends from './pages/Friends';
+
 import Header from './components/Header';
 import Footer from './components/Footer';
 
 export default function App() {
   const [serverMessage, setServerMessage] = useState('');
+  const [watchlistItems, setWatchlistItems] = useState([]);
+  console.log('these are all stored items', watchlistItems);
 
   useEffect(() => {
     fetch('http://localhost:4000/')
       .then((res) => res.json())
       .then((response) => setServerMessage(response));
   });
 
+  function addWatchlistItem(newWatchListItem) {
+    console.log('item to be added to db', newWatchListItem);
+    setWatchlistItems([newWatchListItem, ...watchlistItems]);
+  }
+
   return (
     <div>
       <Header />
 
       <Switch>
         <Route exact path="/">
-          <Main>
-            <h2>Home</h2>
-            <p>{serverMessage}</p>
-          </Main>
+          <Home />
+          <p>{serverMessage}</p>
         </Route>
 
         <Route path="/watchlist">
-          <Main>
-            <h2>Meine Merkliste</h2>
-            <p>{serverMessage}</p>
-          </Main>
+          <Watchlist onAddWatchlistItem={addWatchlistItem} />
+          {watchlistItems.map((item) => (
+            <div>
+              <p>{item.title}</p>
+              <p>{item.category}</p>
+            </div>
```

Comment:
React always needs a key when mapping over items so that it can which items where updated / where changed [React Keys](https://reactjs.org/docs/lists-and-keys.html#keys)
```suggestion
          {watchlistItems.map((item, index) => (
            <div key={index}>
              <p>{item.title}</p>
              <p>{item.category}</p>
            </div>
```

Reply:
adjusted this